t/find.t

```diff
@@ -104,17 +104,17 @@ sub check_find2perl {
 	my( $find2perl ) = @_;
 
 	ok( -e $find2perl, "find2perl exists at $find2perl" );
-	ok( -x $find2perl, "find2perl is executable $find2perl" );
+	ok( -x $find2perl, "find2perl is executable $find2perl" ) if $^O ne 'MSWin32';
```

Comment:
Can you use SKIP blocks where instead?

Reply:
Done. I also unified the nearly-identical `check_find{,2perl`.